index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+
+</head>
+
+<body>
+
+    <header id="header">
+        <nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light navbar-fixed-top">
+            <a class="navbar-brand " href="#"><img src="logo.png" id="header-img" class="img-fluid" alt=""></a>
+
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+                aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item active">
+                        <a class="nav-link text-dark display-8 " href="#">Home </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark display-8" href="#form">Mail</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark display-8" href="#">Pricing</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+    </header>
+
+    <br>
+
+
+    <div id="video" class="embed-responsive embed-responsive-16by9 ml-5">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/wf4F2-9UXUo?autoplay=1"
+            title="YouTube video player" frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen></iframe>
+    </div>
```

Comment:
I think it would be better to use semantic tags. Try to put your all div tags in a section tag
like this
```suggestion
<section>
    <div id="video" class="embed-responsive embed-responsive-16by9 ml-5">
        <iframe width="560" height="315" src="https://www.youtube.com/embed/wf4F2-9UXUo?autoplay=1"
            title="YouTube video player" frameborder="0"
            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
            allowfullscreen></iframe>
    </div>
   </section>
```